docs/api/app.md

```diff
@@ -725,10 +725,27 @@ Returns `string` - User operating system's locale two-letter [ISO 3166](https://
 
 ### `app.getSystemLocale()`
 
-Returns `string` - The current system locale. On Windows and Linux, it is fetched using Chromium's `i18n` library. On macOS, the `NSLocale` object is used instead.
+Returns `string` - The current system locale. On Windows and Linux, it is fetched using Chromium's `i18n` library. On macOS, `[NSLocale currentLocale]` is used instead. To get the user's current system language, which is not always the same as the locale, it is better to use `app.getPreferredSystemLanguages()` and take the first result.
+
+Note that the system locale is not always the most preferred system language:
+
+* On Windows 11, the locale returned is the one used for the regional format. For example, if the regional format is `zh-CN` and the most preferred system language is `fr-FR`, then `zh-CN` is returned.
+* On macOS Monterey, the locale is a string containing a mix of the most preferred system language and the region. For example, if the most preferred language is `pt-BR` but the region is Finland, the locale is `pt-FI` instead of `pt-BR`.
+
+Different operating systems also use the regional data differently:
+
+* Windows 11 uses the regional format to render dates and times.
+* macOS Monterey uses the region for formatting numbers, selecting the currency symbol to use, and rendering dates and times.
```

Comment:
This still does not give a clarity on when a user would prefer to use `app.getSystemLocale` over `app.getPreferredSystemLanguages` and vice-versa. Can you provide an example of how an application could use it.